sorc/grid_tools.fd/regional_esg_grid.fd/pmat.f90

```diff
@@ -113,36 +118,35 @@ subroutine sinvmtf(a,ff)!                                                [inv]
    print '(" In sinvmtf; failed call to sldumf")'
    return
 endif
-
```

Comment:
Why did you remove these blank lines?

Reply:
The blank line here is inconsistent for the coding style. I can add it back if you want.

Reply:
I am in favor of consistent style, but it will not be achieved until we agree what it is, and the provide some automatic formatting. However, that's not what we are doing now. At the moment, doxygen changes only in these doxygen PRs.